### DISCUSSION TITLE
model-viewer on iPhone XR only stable up to 3.0.0

### DISCUSSION THREAD

**Discussion by NahuelGerthVK** [google/model-viewer#4177] [Q&A] [answered]
### Description
The current version of model-viewer on is not working on iPhone XR. The models are simply not displayed (even on the main page: modelviewer.dev). The latest working version is 3.0.0. The console shows no error or warning though.

#### Live Demo
https://modelviewer.dev/

### Version
- working: model-viewer: v3.0.0
- not working: model-viewer: v3.0.1
- not working: model-viewer: v3.0.2

### Browser Affected
- [x] Safari

### OS
- [x] iOS 14.2 on iPhone XR

![IMG_9613](https://user-images.githubusercontent.com/35173372/226703612-c9a78939-89c8-4a61-8a6b-6f3a0873e2ef.PNG)

**Comment by elalish** [comment]
Ugh, I don't have an iPhone XR to test on. Does it work if you upgrade to iOS 15?

**Comment by elalish** [comment]
And that's especially weird because there were so few changes from 3.0.0 to 3.0.1. Any more detail you can give would be helpful - does this happen on any other devices/browsers?

**Comment by NahuelGerthVK** [comment]
I can make an iOS upgrade and see if it works :-) just realised how outdated my OS is.

**Comment by danilokrulj** [comment]
I have an iPhone XR (15.6.1) and its working with the latest version of model-viewer.

**Comment by NahuelGerthVK** [comment, marked as answer]
Ok updating to 16.3.1 also solved this for me

**Comment by SteveX** [comment]
I've just experienced the same issue with my models on 3.0.1 and also the model viewer website. iOS 16.3.1, iPhone 13 Pro. Safari and also Chrome.

Console logs THREE.WebGLRenderer: Context Lost.

The models and the model viewer website were all working on the same phone around a month ago.

I've just updated to 16.4 and everything is working again Safari, but still not loading in Chrome.

**Reply by elalish** [reply]
@mrdoob Have you seen any reports like this recently? I'm tempted to say it's just a random WebKit regression and ignore it, but curious if there's some known workaround...

**Reply by mrdoob** [reply]
I don't think I have no.
